doc/arkode/guide/source/Constants.rst

```diff
@@ -78,6 +78,8 @@ contains the ARKODE output constants.
    +-----------------------------------------------+------------------------------------------------------------+
    | :index:`ARKODE_FORWARD_EULER_1_1`             | Use the Forward-Euler-1-1 ERK method.                      |
    +-----------------------------------------------+------------------------------------------------------------+
+   | :index:`ARKODE_RALSTON_3_1_2`                 | Use the Ralston-3-1-2 ERK method.                          |
+   +-----------------------------------------------+------------------------------------------------------------+
```

Comment:
These methods are generally listed in increasing order of accuracy, so I'd move `ARKODE_RALSTON_3_1_2` further down the list.  We might also want to adjust the order of the ones at the end of the section, and add the method/embedding orders of accuracy to the table text.

Reply:
It's after the first order method and with the other second order methods. I put it first among the second order since it's the new default. We should settle on an unambiguous ordering so, then I can rearrange the table and the Butcher table docs to be consistent.

Reply:
I like that idea, but unless we want anything more detailed that name and order, I think we should save that for later.

Reply:
Agreed, I was thinking this could be the start of such a table